src/assets/scss/styles.scss

```diff
@@ -36,3 +36,9 @@
 .footer-content {
 	padding: 2rem;
 }
+
+.container > h1 {
```

Comment:
Should be part of `PageHeading` component imho

Reply:
Cannot use `media-breakpoint-down` inside scoped SCSS, creating CSS class instead